source/IntegrationTests/Behaviours/IncomingRequests/GivenAggregatedMeasureDataRequestWithDelegationTests.cs

```diff
@@ -0,0 +1,926 @@
+        // It is very important that the generated data is correct,
+        // since (almost) all assertion after this point is based on this data
+        var rejectedMessage = AggregatedTimeSeriesResponseEventBuilder
+            .GenerateRejectedFrom(message.AggregatedTimeSeriesRequest);
+
+        await GivenAggregatedMeasureDataRequestRejectedIsReceived(message.ProcessId, rejectedMessage);
+
+        // Act
+        var originalActorPeekResults = await WhenActorPeeksAllMessages(
+            originalActor.ActorNumber,
+            originalActor.ActorRole,
+            peekDocumentFormat);
+
+        var delegatedActorPeekResults = await WhenActorPeeksAllMessages(
+            delegatedToActor.ActorNumber,
+            delegatedToActor.ActorRole,
+            peekDocumentFormat);
+
+        // Assert
+        PeekResultDto peekResult;
+        using (new AssertionScope())
+        {
+            originalActorPeekResults.Should().BeEmpty("because original actor shouldn't receive result when delegated actor made the request");
+            peekResult = delegatedActorPeekResults.Should().ContainSingle("because there should only be one rejected message regardless of grid areas")
+                .Subject;
+
+            peekResult.Bundle.Should().NotBeNull("because peek result should contain a document stream");
+        }
+
+        var expectedReasonMessage = "Det er kun muligt at anmode om data på for en hel måned i forbindelse"
+                                    + " med en balancefiksering eller korrektioner / It is only possible to request"
+                                    + " data for a full month in relation to balancefixing or corrections";
+
+        await ThenRejectRequestAggregatedMeasureDataDocumentIsCorrect(
+            peekResult.Bundle,
+            peekDocumentFormat,
+            new(
+                BusinessReason.BalanceFixing,
+                "5790001330552",
+                delegatedToActor.ActorNumber.Value,
+                InstantPattern.General.Parse("2024-07-01T14:57:09Z").Value,
+                ReasonCode.FullyRejected.Code,
+                "123564789123564789123564789123564787",
+                "E17",
+                expectedReasonMessage));
+    }
+
+    /// <summary>
+    /// Even though an actor has delegated his requests to another actor, he should still
+    /// be able to request and receive his own data
+    /// </summary>
+    [Theory]
+    [MemberData(nameof(DocumentFormatsWithRoleCombinationsForNullGridArea), MemberType = typeof(GivenAggregatedMeasureDataRequestWithDelegationTests))]
+    public async Task AndGiven_DelegationInTwoGridAreas_AndGiven_OriginalActorRequestsOwnDataWithDataInTwoGridAreas_When_OriginalActorPeeksAllMessages_Then_OriginalActorReceivesTwoNotifyAggregatedMeasureDataDocumentWithCorrectContent(DocumentFormat incomingDocumentFormat, DocumentFormat peekDocumentFormat, ActorRole delegatedFromRole, ActorRole delegatedToRole)
```

Comment:
```suggestion
    public async Task AndGiven_DelegationInTwoGridAreas_AndGiven_OriginalActorRequestsOwnDataWithDataInThreeGridAreas_When_OriginalActorPeeksAllMessages_Then_OriginalActorReceivesTwoNotifyAggregatedMeasureDataDocumentWithCorrectContent(DocumentFormat incomingDocumentFormat, DocumentFormat peekDocumentFormat, ActorRole delegatedFromRole, ActorRole delegatedToRole)
```
two -> three
Mocking response corresponding to 3 grid areas